docs/input/docs/installation/configure.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
I would move this under the Getting Started section

Reply:
Moved